### ISSUE TITLE
Unsent messages are transfered between channels

### ISSUE THREAD

**Issue by danielcybulka** [ZbayApp/zbay#157]
Steps to reproduce:
1. Write any message in any channel but don't send it.
2. Switch channel. Unsent text appears in message creator of new channel.

**Comment by holmesworcester** [comment]
Yeah, expected behavior is that the message remains unsent *only in the channel for which you were composing it*, at least until app is restarted. so you could potentially have multiple unfinished "drafts" awaiting sending in multiple channels.

**Comment by NorbertBodziony** [comment]
Right it seems like a lot of refactoring maybe we can postpone it to do it during lite wallet integration?

**Comment by holmesworcester** [comment]
Okay let's postpone.

**Comment by NorbertBodziony** [comment]
#223 connected.
